### PR TITLE
[WIP] Integration bitmovin player

### PR DESCRIPTION
**Pull request by CSeal**
WIP: added supporting of subtitles: native android part, basic react-native control interface. has problems with showing subtitles view. improved performance and graphic usability of the player's control interface.

additionally,  fixed behavior of the back button for remote control on Android #12.
fixed bug with sending analytics of Bitmovin player #21  

about subtitles problem, when I chose some subtitle track, our player in some time can be damaged with error code 2001 -  "Input not valid"(for example, it occurs, when I chose the track with id like "subtitle_group:en").  I don`t know What is related with

### CODE REVIEW

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java [hunk 1]
```diff
@@ -84,6 +87,10 @@ public PlayerView createViewInstance(ThemedReactContext context) {
     player = Player.create(reactContext, playerConfig);
 
     playerView = new PlayerView(reactContext, player);
+/*     subtitleView = new SubtitleView(reactContext, null);
```
Comment (tomdyson) [inline android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:90]: Unless this is a WIP PR, I don't think we should commit commented-out code

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java [hunk 2]
```diff
@@ -212,9 +224,9 @@ public void setConfiguration(PlayerView view, ReadableMap config) {
     if (configuration != null && configuration.getString("url") != null) {
       url = configuration.getString("url");
 
-      if (configuration.getString("subtitles") != null) {
-        subtitleTrack = new SubtitleTrack(configuration.getString("subtitles"), null, "en", "en", false, "en");
-      }
+      //if (configuration.getString("subtitles") != null) {
```
Comment (tomdyson) [inline android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:227]: As above

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java [hunk 3]
```diff
@@ -224,7 +236,7 @@ public void setConfiguration(PlayerView view, ReadableMap config) {
         poster = configuration.getString("poster");
       }
 
-      List subtitleTracks = Arrays.<SubtitleTrack>asList(subtitleTrack);
+      //List subtitleTracks = Arrays.<SubtitleTrack>asList(subtitleTrack);
```
Comment (tomdyson) [inline android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:239]: As above

File: src/screens/EventDetailsScreen/index.tsx [hunk 4]
```diff
@@ -73,13 +73,25 @@ const EventDetailsScreen: React.FC<TEventDetalsScreenProps> = ({ route }) => {
       <Player
         autoPlay
         configuration={{
-          url: 'https://video-ingestor-output-bucket.s3-eu-west-1.amazonaws.com/12338/manifest.m3u8',
+          url: 'https://video-ingestor-output-bucket.s3.eu-west-1.amazonaws.com/6565/manifest.m3u8', // https://video-ingestor-output-bucket.s3-eu-west-1.amazonaws.com/12338/manifest.m3u8
```
Comment (tomdyson) [inline src/screens/EventDetailsScreen/index.tsx:76]: As above

**Review by tomdyson — COMMENTED** [review]
Thanks for this, @CSeal. I'm not qualified to review the Java or RN changes, but I'd like us to avoid committing commented-out code - let's just use git history for that.

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java (hunk 1 above)
Comment (CSeal) [reply android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:90]: it is a WIP, about subtitles. I will remove it later.

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java (hunk 2 above)
Comment (CSeal) [reply android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:227]: it is a WIP, about subtitles. I will remove it later.

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java (hunk 3 above)
Comment (CSeal) [reply android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:239]: it is a WIP, about subtitles. I will remove it later.

File: src/screens/EventDetailsScreen/index.tsx (hunk 4 above)
Comment (CSeal) [reply src/screens/EventDetailsScreen/index.tsx:76]: it is a WIP, about subtitles. I will remove it later.

File: android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java (hunk 1 above)
Comment (tomdyson) [reply android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java:90]: OK, thanks, I've updated the PR title and converted it to a draft, to make this clear.

**Comment by CSeal** [comment]
cleared unnecessary comments

**Pull request merged**